integration_tests/e2e/overview/educationAndTrainingTab.cy.ts

```diff
@@ -354,7 +354,7 @@ context('Prisoner Overview page - Education And Training tab', () => {
   describe('should display a link to view all courses and qualifications', () => {
     it(`should link to the courses and qualifications page`, () => {
       // Given
-      cy.task('stubGetInductionShortQuestionSet')
+      cy.task('stubLearnerEducationWithCoursesQualificationsCompletedInLast12Months')
```

Comment:
This cypress test might need some love in a future PR as I'm not convinced either the test or the implementation is 100% correct. The change I've made to the stub here is enough to keep it passing, but I think we'll be looking at this test again in the near future.